projects/plugins/jetpack/modules/videopress/class.videopress-ajax.php

```diff
@@ -72,13 +72,19 @@ private function is_valid_guid( $guid ) {
 	 * @return void
 	 */
 	public function wp_ajax_videopress_get_playback_jwt() {
-		$guid = filter_input( INPUT_POST, 'guid' );
+		$guid             = filter_input( INPUT_POST, 'guid' );
+		$embedded_post_id = filter_input( INPUT_POST, 'post_id', FILTER_VALIDATE_INT );
+
+		if ( false === $embedded_post_id ) {
```

Comment:
filter_input may also return null, you could either check empty( $embedded_post_id ) or use the options array with default => 0 like

`$embedded_post_id = (int) filter_input( INPUT_POST, 'post_id', FILTER_VALIDATE_INT, array( 'options' => array( 'default' => 0 ) );`